Src/AutoFixture.NUnit3.UnitTest/DependencyConstraints.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Linq;
+using NUnit.Framework;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class DependencyConstraints
+    {
+        [TestCase("Moq")]
+        [TestCase("Rhino.Mocks")]
+        public void AutoFixtureNUnit3DoesNotReference(string assemblyName)
+        {
+            // Fixture setup
+            // Exercise system
+            var references = typeof(AutoDataAttribute).Assembly.GetReferencedAssemblies();
+            // Verify outcome
+            Assert.False(references.Any(an => an.Name == assemblyName));
+            // Teardown
+        }
+
+        [TestCase("Moq")]
+        [TestCase("Rhino.Mocks")]
```

Comment:
I ported this over from **AutoFixture.NUnit2**, I guess could it be that **Moq** and **Rhino.Mocks** are NUnit specific and the others are not?

If that's not the case maybe an issue should be started for **AutoFixture.NUnit2** too?